src/yio-plugin/integration.h

```diff
@@ -100,5 +100,9 @@ class Integration : public QObject, public IntegrationInterface {
     YioAPIInterface*        m_yioapi;
     ConfigInterface*        m_config;
     QLoggingCategory&       m_logCategory;
-    QStringList             m_allAvailableEntities;
+    QVariantList            m_allAvailableEntities;
+    /// containts QVariantMap objects with the following keys
+    /// "entity_id": entity id
+    /// "type": type of entity, like light
+    /// "integration_id": id of the integration controlling the entity
```

Comment:
I don't understand those comments. Are they related to the content of QVariantList?
Then the description must be added to the setter method.